tests/integration/session/test_timeout.py

```diff
@@ -32,12 +30,3 @@ def test_when_session_times_out_server_side_401_is_returned(self):
         time.sleep(3)
         self.get(self.last_url)
         self.assertStatusUnauthorised()
-
-    def test_schema_defined_timeout_is_used(self):
-        self._application.config['EQ_SESSION_TIMEOUT_SECONDS'] = 10
-        self.launchSurvey('test', 'timeout')
-        self.assertInHead('window.__EQ_SESSION_TIMEOUT__ = 6')
-
-    def test_schema_defined_timeout_cant_be_higher_than_server(self):
```

Comment:
Quick point, unless I missed somehting obvious, these would still be valid tests.
*Ensuring schema defined timeout cannot be longer than app default is useful.*